ansible/playbooks/ubuntu.yml

```diff
@@ -53,16 +53,21 @@
         - ccache
         - cpanminus
         - cpio
+        - curl
         - flex
         - g++
         - gcc
+        - g++-4.8
+        - gcc-4.8
         - git
         - libasound2-dev
         - libcups2-dev
         - libdwarf-dev
         - libelf-dev
         - libexpat1-dev
         - libffi-dev
+        - libfreetype6-dev
+        - libnuma-dev
```

Comment:
This package "libnuma-dev" is not available on arm 32bit - armv7l or at least not in the default repos.
(raspbian jessie)

      when:
        - ansible_architecture != "armv7l"